### PR TITLE
new validators

### PR DESCRIPTION
**Pull request by kehshiba**

### CODE REVIEW

File: validators/Aadhar_IN.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
```
Comment (SwapnilSoni1999) [inline validators/Aadhar_IN.js]: 1. Aadhar card number can be in string or number
2. Aadhar length needs to be checked
3. `x` should be overwritten with to use `.length` and `.match` in future execution
4. `typeof` check needs to return the message of the data type we're expecting

File: validators/DRIVING_LC_IN.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
+        return { success: false, message: 'Invalid Driving License' }
```
Comment (SwapnilSoni1999) [inline validators/DRIVING_LC_IN.js:5]: - typeof check should return the data type we're expecting

File: validators/PAN_IN.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
+        return { success: false, message: 'Invalid PAN Number' }
```
Comment (SwapnilSoni1999) [inline validators/PAN_IN.js]: - again, type check should return the data type we're expecting

**Review by SwapnilSoni1999 — CHANGES_REQUESTED** [review]
Please do the following changes!

**Comment by SwapnilSoni1999** [comment]
Also please rebase your commits in proper messages for seperate files

**Comment by kehshiba** [comment]
hey i did some changes?

**Comment by kehshiba** [comment]
do i have to change the message text? also i'm learning about how to rebase now but i commited though?

**Comment by SwapnilSoni1999** [comment]
> do i have to change the message text? also i'm learning about how to rebase now but i commited though?

You can learn more about how to rebase your commits from github docs

**Comment by SwapnilSoni1999** [comment]
Since the issue is resolved by #8 and #9 im closing this

**Pull request closed without merging**